docs/UserGuide.md

```diff
@@ -89,6 +105,7 @@ Format: `list`
 ### Viewing help: `help`
 
 Shows commonly used commands for **_TBM_** in a new help window. Pressing Esc will close this window.
+![help window](./images/command-screenshots/help_window.png)
```

Comment:
Should this be moved to the appendix as well? Then a link to it with "new help window". It seems out of place now compared to the other commands,